src/OSPSuite.InstallationValidator.Core/InstallationValidatorConfiguration.cs

```diff
@@ -42,6 +44,36 @@ private string applicationSettingsFolderPathFor(string applicationFolderPath)
          return Path.Combine(EnvironmentHelper.ApplicationDataFolder(), applicationFolderPath);
       }
 
+      public string PKSimBinaryExecutablePath
```

Comment:
Let's create a method for all those guys. Code is exactly the same except for the actual REgEntry and RegValue